src/navigation/IdeaNavigator.js

```diff
@@ -33,13 +33,13 @@ const linking = {
 
 export const IdeaNavigator = () => {
   const { t } = useTranslation()
-  const { loading, user } = useAuthorization()
```

Comment:
@goapunk now it throws an error and the profileContext cannot set a user (for username, userId and userImage).